misc/autodoc/autodoc.py

```diff
@@ -164,8 +161,6 @@ def add_page(title, text):
 
 if __name__ == '__main__':
     args = parse_args()
-    logging.info("building planner...")
-    build_planner(args.build)
```

Comment:
Has the buildbot been adapted for this change, or will this break our wiki?

Reply:
Good point. Looking at master.cfg in the infrastructure repo, it seems like the worker first compiles the planner and then runs autodoc: 
```
factory_dict["autodoc"] = util.BuildFactory([
    update_step(incremental=True),
    autodoc_step,
])
```
I'm not sure, though.

Reply:
Yes, that's the right file. The `update_step` is just to pull from the repository. If you look at the other build factories above, they all include `build_all_step`, which does the actual build.